bot/storage-mongo/src/main/kotlin/FeatureMongoDAO.kt

```diff
@@ -72,17 +76,33 @@ internal class FeatureMongoDAO(private val cache: FeatureCache, private val col:
             }
         } else {
             val feature = (connectorFeature ?: globalFeature)!!
-            isEnabled(feature)
+            isEnabled(feature, userId)
         }
     }
 
-    private fun isEnabled(feature: Feature): Boolean {
-        val now = ZonedDateTime.now(internalDefaultZoneId)
+    private fun isEnabled(feature: Feature, userId: String?): Boolean {
+        return feature.enabled
+                && isEnableForDate(feature, ZonedDateTime.now(internalDefaultZoneId))
+                && isEnableForUser(feature, userId)
+    }
+
+    private fun isEnableForUser(feature: Feature, userId: String?): Boolean {
```

Comment:
Here you have to set a a default value for userId, to ensure backward compatibility : `userId: String? = null`